sources/packages/web/src/views/supporting-user/SupportingInformation.vue

```diff
@@ -91,6 +134,53 @@ export default {
     const studentsLastName = ref("");
     const studentsDateOfBirth = ref();
     const initialData = ref();
+    const formioUtils = useFormioUtils();
+    const isFirstPage = ref(true);
+    const isLastPage = ref(false);
+    const showNav = ref(false);
+    let formInstance: any;
+
+    const wizardSubmit = () => {
+      formInstance.submit();
+    };
+
+    const formLoaded = async (form: any) => {
+      showNav.value = true;
+      // Emit formLoadedCallback event to the parent, so that parent can
+      // perform the parent specific logic inside parent on
+      // form is loaded
+      context.emit("formLoadedCallback", form);
+      formInstance = form;
+      // Disable internal submit button.
+      formioUtils.disableWizardButtons(formInstance);
+      formInstance.options.buttonSettings.showSubmit = false;
+      // Handle the navigation using the breadcrumbs.
+      formInstance.on("wizardPageSelected", (page: any, index: number) => {
```

Comment:
It seems that page is not used here, it can be "omitted" as shown below.
```ts
formInstance.on("wizardPageSelected", (_: unknown, index: number)
```

Reply:
That's right. Thanks!

Reply:
Was it changed, I am not able to see the file changed.

Reply:
yes, please check it again.